python/integrationtests-basic

```diff
@@ -42,11 +45,23 @@ class SimulationTestCase(unittest.TestCase):
         for o in self.stream_iter:
             if o.name == 'UAVO_HwSimulation':
                 if o.FakeTickBlocked != 0:
+                    self.tick_num += 1
                     return True
 
         # This is bad.  Got to end of stream, no tick.
         return False
 
+    def send_control_values(self, values):
+        val_tup = tuple(values)
+
+        print("send_control_values", self.tick_num, val_tup)
+
+        gi = self.uavrcvr_class._make_to_send(val_tup)
```

Comment:
Should make_to_send actually be private API? It seems to be one of the most used methods. :stuck_out_tongue: